src/lin_diffEq_HHL.jl

```diff
@@ -19,106 +19,91 @@ export QuEuler, QuLeapfrog, QuAB2, QuAB3, QuAB4
 """
 
 """
-    alg_lin_diffeq
+    AlgQuDiff
     * step - step for multistep method
     * α - coefficients for xₙ
     * β - coefficent for xₙ'
 """
-struct alg_lin_diffeq
-  step::Int
-  α::Array
-  β::Array
-  alg_lin_diffeq(step,α,β) = new(step,α,β)
+
+struct AlgQuDiff
+    step::Int
+    α::Array
+    β::Array
+    AlgQuDiff(step,α,β) = new(step,α,β)
```

Comment:
There's no need to define this constructor, this is just the default constructor.